src/app/public/modules/phone-field/phone-field.component.html

```diff
@@ -1,45 +1,96 @@
 <div
   class="sky-form-group sky-input-group sky-phone-field"
+  [@initalLoad]
 >
   <div
     class="sky-input-group-btn sky-phone-field-country-btn"
   >
-    <sky-dropdown
+    <button
+      aria-live="polite"
+      class="sky-btn sky-btn-default"
+      [attr.title]="('skyux_phone_field_country_select_label' | skyAppResources) + ' ' + ('skyux_phone_field_country_selected_label' | skyAppResources:selectedCountry.name)"
+      [attr.aria-label]="('skyux_phone_field_country_select_label' | skyAppResources) + ' ' + ('skyux_phone_field_country_selected_label' | skyAppResources:selectedCountry.name)"
       [disabled]="countrySelectDisabled"
-      [label]="'skyux_phone_field_dropdown_label' | skyLibResources"
-      [title]="selectedCountry.name + ' +' + selectedCountry.dialCode"
+      (click)="toggleCountrySearch(true)"
     >
-      <sky-dropdown-button>
-        <div
-          [className]="'iti-flag ' + selectedCountry.iso2"
+      <div
+        [className]="'iti-flag ' + selectedCountry.iso2"
+      >
+      </div>
+    </button>
+  </div>
+  <ng-container *ngIf="phoneInputShown">
+    <span
+      class="sky-phone-field-container"
+      [@phoneInputAnimation]
+      (@phoneInputAnimation.done)="phoneInputAnimationEnd($event)"
+    >
+      <ng-content></ng-content>
+    </span>
+  </ng-container>
+  <ng-container *ngIf="countrySearchShown">
+    <span
+      class="sky-phone-field-container"
+      [@countrySearchAnimation]
+      (@countrySearchAnimation.done)="countrySearchAnimationEnd($event)"
+
+    >
+      <form
+        novalidate
+        class="sky-phone-field-country-search"
+        [formGroup]="countrySearchForm"
+      >
+        <sky-autocomplete
+          [data]="countries"
+          [propertiesToSearch]="['name', 'iso2']"
+          [searchResultTemplate]="countrySearchResultTemplate"
+          [searchTextMinimumCharacters]="2"
```

Comment:
Was `2` a suggestion from UX? I noticed when searching for `ma` you can get so many results it triggers a full screen modal. Think that's ok?

Reply:
We have to allow for 2 because we are allowing for the searching using the ISO country code. That is a 2 letter code so we can't go above that. I believe we were originally going to do 3 but had to drop it to 2 for the country code search.

Reply:
Cool, that's a great reason!